src/Utilities.Mongo/Interfaces/IMongoContext.cs

```diff
@@ -0,0 +1,25 @@
+﻿using MongoDB.Driver;
+using System.Threading.Tasks;
+
+namespace MatrTech.Utilities.Mongo.Interfaces
+{
+    public interface IMongoContext
+    {
+        /// <summary>
+        /// Checks to see if a collection exists within the context.
```

Comment:
Checks whether a collection with the name <param name="collectionName"> exists within the context.